src/components/Icon/BankIcons.js

```diff
@@ -0,0 +1,122 @@
+import SunTrust from '../../../assets/images/bankicons/suntrust.svg';
+import TdBank from '../../../assets/images/bankicons/td-bank.svg';
+import USBank from '../../../assets/images/bankicons/us-bank.svg';
+import USAA from '../../../assets/images/bankicons/usaa.svg';
+
+/**
+ * Returns matching asset icon for bankName
+ * @param {String} bankName
+ * @param {Boolean} isCard
+ * @returns {Object}
+ */
+
+function getAssetIcon(bankName, isCard) {
+    if (bankName.includes('americanexpress')) {
+        return AmericanExpress;
+    }
+
+    if (bankName.includes('bank of america') || bankName.includes('bankofamerica')) {
+        return BankOfAmerica;
+    }
+
+    if (bankName.startsWith('bbt')) {
+        return BB_T;
+    }
+
+    if (bankName.startsWith('capital one') || bankName.includes('capitalone')) {
+        return CapitalOne;
+    }
+
+    if (bankName.startsWith('chase') || bankName.includes('chase')) {
+        return Chase;
+    }
+
+    if (bankName.includes('charles schwab') || bankName.includes('charlesschwab')) {
+        return CharlesSchwab;
+    }
+
+    if (bankName.startsWith('citibank') || bankName.includes('citibank')) {
+        return CitiBank;
+    }
+
+    if (bankName.startsWith('citizens bank') || bankName.includes('citizensbank')) {
+        return CitizensBank;
+    }
+
+    if (bankName.startsWith('discover ') || bankName.includes('discover.') || bankName === 'discover') {
+        return Discover;
+    }
+
+    if (bankName.startsWith('fidelity')) {
+        return Fidelity;
+    }
+
+    if (bankName.startsWith('huntington bank') || bankName.includes('huntingtonnational') || bankName.includes('huntington national')) {
+        return HuntingtonBank;
+    }
+
+    if (bankName.startsWith('navy federal credit union') || bankName.includes('navy federal credit union')) {
+        return NavyFederalCreditUnion;
+    }
+
+    if (bankName.startsWith('pnc') || bankName.includes('pnc')) {
+        return PNC;
+    }
+
+    if (bankName.startsWith('regions bank') || bankName.includes('regionsbank')) {
+        return RegionsBank;
+    }
+
+    if (bankName.startsWith('suntrust') || bankName.includes('suntrust')) {
+        return SunTrust;
+    }
+
+    if (bankName.startsWith('td bank') || bankName.startsWith('tdbank') || bankName.includes('tdbank')) {
+        return TdBank;
+    }
+
+    if (bankName.startsWith('us bank') || bankName.startsWith('usbank')) {
+        return USBank;
+    }
+
+    if (bankName.includes('usaa')) {
+        return USAA;
+    }
+
+    return isCard ? CreditCard : Bank;
+}
+
+/**
+ * Returns Bank Icon that matches to existing bank icons or default icons
+ * @param {String} bankName
+ * @param {Boolean} [isCard = false]
+ * @returns {Object}
+ */
+
+export default function getBankIcon(bankName, isCard) {
+    if (!bankName) {
+        // Returns default card or bank icon, if bankName is missing
+        return isCard ? CreditCard : Bank;
+    }
+
+    if (bankName === 'Expensify Card') {
```

Comment:
Sorry for being unclear, can you move this into the top of `getAssetIcon`

Reply:
Then would check be like `bankName === 'expensify card'`.

Regarding I'll provide mock screenshots on all devices, I don't have real-time data can you verify that?

Reply:
> Then would check be like bankName === 'expensify card'.

Sounds good!

> Regarding screenshots, I'll provide mock screenshots on all devices, I don't have real-time data can you verify that?

I only have a couple of accounts on prod, but LGTM!

![2021-09-02_10-56-57](https://user-images.githubusercontent.com/42391420/131885898-963246c9-5e5d-4932-b85f-2424e9ec458f.png)

Reply:
![image](https://user-images.githubusercontent.com/85645967/131887309-c4baded7-1d66-462f-9002-edc17e0bcf60.png)

Need to resize the Default Bank, Card and Expensify  as shown above as per @shawnborton suggestion

Do you feel otherwise?

Reply:
Refer comments
https://github.com/Expensify/App/pull/4653#issuecomment-911220887
https://github.com/Expensify/App/pull/4653#issuecomment-911441982
https://github.com/Expensify/App/pull/4653#issuecomment-911455402
https://github.com/Expensify/App/pull/4653#issuecomment-911473261

Reply:
Based on your final comment I'll make changes & push if needed or just update the PR tests and add screenshots. Thanks
@stitesExpensify

Reply:
> Need to resize the Default Bank, Card and Expensify as shown above as per @shawnborton suggestion
> Do you feel otherwise?

What shawn says goes when it comes to design :smile: Please resize them in addition to moving the `bankName === 'expensify card'`

Reply:
Sure, will update PR by tomorrow!

Reply:
updated!